packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -164,9 +164,9 @@ abstract contract StandardBridge {
     }
 
     /**
-     * @notice Allows EOAs to deposit ETH by sending directly to the bridge.
+     * @notice Allows EOAs to bridge ETH by sending directly to the bridge.
```

Comment:
The one question is whether or not we want to delete the implementation here and just leave the virtual function def

Reply:
I lean towards yes here

Reply:
+1, make this virtual

Reply:
I'm not quite sure what's been agreed to here, but I feel strongly that we should not have an implementation here which is never used and gets overridden everywhere it's inherited. 

Let's just do this please:

`receive() external payable virtual;`

Reply:
This is what we were agreeing on